gating/pre_merge_test/post_send_junit_to_qtest.sh

```diff
@@ -10,6 +10,10 @@ export QTEST_API_TOKEN=$RPC_ASC_QTEST_API_TOKEN
 VENV_NAME="venv-qtest"
 PROJECT_ID="76551"
 
+# Work-around for ASC-592. Hardcoded for proper results in qtest
+export RPC_PRODUCT_RELEASE="newton"
+export RPC_RELEASE="r13.?.?"
```

Comment:
Remove this line and place the following in the `gating/pre_merge_test/run_deploy_mnaio.sh` file in order to dynamically set the value for `RPC_RELEASE`:
```
## Functions -----------------------------------------------------------------
export BASE_DIR=${BASE_DIR:-"/opt/rpc-openstack"}
source ${BASE_DIR}/scripts/functions.sh
```